examples/platform/silabs/SiWx917/BaseApplication.cpp

```diff
@@ -296,6 +318,122 @@ void BaseApplication::LightEventHandler()
     // the LEDs at an even rate of 100ms.
     //
     // Otherwise, blink the LED ON for a very short time.
+    if (mFunction != kFunction_FactoryReset)
```

Comment:
From what i see, This whole if for the identify behaviour doesn't do anything if ENABLE_WSTK_LEDS is not define. Instead of having 10 of those ifdef, just wrap the whole if with a single one

Reply:
sure